### PR TITLE
Revert PR 5035 which appears to be causing e2e failures

### PR DESCRIPTION
**Pull request by jmthibault79**
This reverts commit 59ca1426313c1cc3e4f7a943ca53b9c9a92eae6b.

Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

**Review by calbach — APPROVED** [review]
Wow, passing - I haven't gotten it to pass all day, I guess this is somehow related though I don't immediately see the connection. I suppose it could be related to how Javascript gets loaded onto the page, which could affect the Puppeteer interactions.

File: ui/package.json [hunk 1]
```diff
@@ -178,9 +178,6 @@
     "tslint-microsoft-contrib": "^6.0.0",
     "typescript": ">=3.2.1 <3.3.0",
     "wd": "^1.12.1",
-    "webpack-dev-server": "^3.11.2"
-  },
-  "resolutions": {
-    "webpack-dev-server": "^3.11.2"
```
Comment (jmthibault79) [inline ui/package.json:184]: I wonder if this was the culprit?  [WDS] = webpack-dev-server?

I'll see if isolating the revert to this change also works.

Comment (jmthibault79) [reply ui/package.json:184]: Isolated PR is #5084

**Comment by jmthibault79** [comment]
Closing in favor of #5084

**Pull request closed without merging**